internal/store/node.go

```diff
@@ -90,6 +90,7 @@ func createNodeInfoFamilyGenerator() generator.FamilyGenerator {
 				"kubeproxy_version",
 				"provider_id",
 				"pod_cidr",
+				"system_uuid",
```

Comment:
why this is needed?

Reply:
We report this field in Api server mode, hence need this in KSM mode to maintain parity.